edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CreatePodParameters.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using Microsoft.Azure.Devices.Edge.Agent.Docker.Models;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.Json;
+    using Newtonsoft.Json;
+
+    public class CreatePodParameters
+    {
+        public CreatePodParameters(
+            IList<string> env,
+            IDictionary<string, global::Docker.DotNet.Models.EmptyStruct> exposedPorts,
+            HostConfig hostConfig,
+            string image,
```

Comment:
Please see my comment in the CombinedKubernetesConfig.cs file. I feel this should be a list of images